src/main/java/overlap_crap_fix_locations_later/application_business/TimetableMatchInputBoundary.java

```diff
@@ -0,0 +1,14 @@
+package overlap_crap_fix_locations_later.application_business;
+
```

Comment:
This interface could use a brief explanation of what its methods do.

Reply:
Done! Added documentation with warnings to not use the return values.